projects/npm-tools/packages/npm-scripts/src/utils/createFederationConfig.js

```diff
@@ -9,37 +9,153 @@ const {
 } = require('webpack');
 
 const createTempFile = require('./createTempFile');
+const getMergedConfig = require('./getMergedConfig');
 const parseBnd = require('./parseBnd');
 const writeWebpackFederationEntryPoint = require('./writeWebpackFederationEntryPoint');
 
 /**
- * This object must represent the current configuration status of the portal's
- * project if we don't want to break the build. It contains, for each
- * federation-enabled project, the packages that are federated.
+ * The following two arrays are roughly equivalent to the old default preset
```

Comment:
Turns out that I was wrong in the previous PR and it didn't make too much sense to mix imports and federation because the model is a bit different.